mintpy/prep_fringe.py

```diff
@@ -75,6 +75,14 @@ def create_parser():
     parser.add_argument('-o', '--out-dir', dest='outDir', type=str, default='./mintpy',
                         help='output directory (default: %(default)s).')
 
+    parser.add_argument('-r','--range','-x', dest='lks_x', type=int, default=1,
+                        help='number of multilooking in range  /x direction (default: %(default)s).')
+    parser.add_argument('-a','--azimuth','-y', dest='lks_y', type=int, default=1,
+                        help='number of multilooking in azimuth/y direction (default: %(default)s).')
+
```

Comment:
1. I would recommend not using `-x` and `-y` here because they are more commonly used as to specify the subset range in other scripts.

2. I don't quite get the purpose of this. Could you elaborate?

Reply:
1. Yes, can swith to -r and -a like in multilook.py 
2. Right now it's only for adding the metadata, which I'm finding useful for keeping track of which timeseries corresponds to which experiment. 
- I have a set of bash scripts that prepares the mulitlooked metadata for `prep_fringe`. Eventually I hope to incorporate them so that when you specify looks to `prep_fringe` it will make them on the fly

Reply:
Do you mean that you have some multilooked data files, based on fringe outputs, but then the metadata is not consistent with the data file, so you would like to update them here?

Reply:
Exactly

Reply:
`multilook.py -x -y` looks clear and intuitive to me, as it only does the multilooking, and applies to geocoded files. The same also applies to `subset.py -x -y`. While `prep_fringe.py -x -y` can be ambiguous, as we don't know it's for multilooking or subset. I hope this explains that I am not double standard ;)

Reply:
Thanks for the explanation. That makes sense. Please add a comment to the help msg as well.

Reply:
Thanks no prob!